### PR TITLE
Add Jazzband Contributing Guidelines

### PR DESCRIPTION
**Pull request by hugovk**
For https://github.com/jazzband/tablib/issues/378.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[![Jazzband](https://jazzband.co/static/img/jazzband.svg)](https://jazzband.co/)
+
+This is a [Jazzband](https://jazzband.co/) project. By contributing you agree to abide
+by the [Contributor Code of Conduct](https://jazzband.co/about/conduct) and follow the
+[guidelines](https://jazzband.co/about/guidelines).
+
+If you'd like to contribute, simply fork 
+[the repository](https://github.com/jazzband/tablib), commit your changes to the
+**develop** branch (or branch off of it), and send a pull request. Make sure you add
+yourself to [AUTHORS](https://github.com/jazzband/tablib/blob/master/AUTHORS).
```
Comment (claudep) [inline .github/CONTRIBUTING.md]: I'm not sure we'll continue using that develop branch. Looks like recent development happened in master. What do you think?

**Comment by claudep** [comment]
Could you just prefix the commit with `Refs #378 -` ?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (hugovk) [reply .github/CONTRIBUTING.md]: I agree to use the master branch as the main branch for development, and `develop` can be deleted. I'll update the text.

**Comment by hugovk** [comment]
Prefixed!

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (hugovk) [reply .github/CONTRIBUTING.md]: Updated to say:

> If you'd like to contribute, simply fork the repository, commit your changes to a feature branch, and send a pull request to master.

Comment (claudep) [reply .github/CONTRIBUTING.md]: Created #380 to track the removal of the develop branch.

**Comment by claudep** [comment]
Now about the format, I realized that we used ReST for the README and you suggest Markdown for the CONTRIBUTING. Maybe we should settle on one format? I have no personal preferences.

**Comment by hugovk** [comment]
I strongly prefer Markdown, I find it much easier to use, especially for simpler pages on GitHub. If you have a need for complex docs, then RST may be better.

I can make another PR to convert the README to Markdown, and update `setup.py` for that too.

**Comment by hugovk** [comment]
Please see PR #381.

**Pull request merged**